rmgpy/data/solvation.py

```diff
@@ -324,7 +320,22 @@ def getSolventStructure(self, label):
         Get a solvent's molecular structure as SMILES or adjacency list from its name
         """
         return self.entries[label].item
-        
+
+    def get_structure(self, label, molecule):
```

Comment:
I understand the desire to reduce code duplication, but this really shouldn't be a bound function of SolventLibrary.  I suggest adding a similar function to species.py or just having the code duplication.